presto-native-execution/etc/catalog/iceberg.properties

```diff
@@ -0,0 +1,4 @@
+# The Presto "iceberg" catalog is handled by the hive connector in Presto native execution.
+connector.name=hive
```

Comment:
I think we can add a mapping from `iceberg` name to `hive` on the Velox side. Similar to how hive-hadoop2 is mapped. Not a blocker for now.

Reply:
Created a PR here https://github.com/facebookincubator/velox/pull/8765